src/frontend/github-oauth.ts

```diff
@@ -30,19 +32,29 @@ export default (opts: OAuthOptions): GithubOAuth => {
 
 	logger.info(opts, "Configuring github auth");
 
-	function login(req: Request, res: Response): void {
+	async function login(req: Request, res: Response): Promise<void> {
+
+		const traceLogsEnabled = await booleanFlag(BooleanFlags.TRACE_LOGGING, false);
```

Comment:
Shouldn't we add `jiraHost` to `booleanFlag` call?

Reply:
see comment above